record_api/core.py

```diff
@@ -383,11 +408,30 @@ def __call__(self) -> None:
                 (self.TOS, self.TOS1), BINARY_OPS[opname], (self.TOS1, self.TOS)
             )
 
+        if self.previous_stack and self.previous_stack.opname in FUNCTION_CALL_OP_NAMES:
+            self.log_called_method()
+
         method_name = f"op_{opname}"
         if hasattr(self, method_name):
             getattr(self, method_name)()
         return None
 
+    def log_called_method(self):
+        if self.previous_stack.log_call_args:
+            tos = self.TOS
+            if type(tos) is type and issubclass(tos, Exception):
```

Comment:
Could we try to add a test for the exception handling, if possible?

Reply:
Good idea.